### PR TITLE
feat(deb): add `priority` and `section` options

### PR DESCRIPTION
**Pull request by naman-crabnebula**
I am adding some necessary `config` options for debian packages.

These should be accessible via `[package.metadata.packager.deb]`

These changes are running perfectly fine on Arch Linux as well as Debian, but please, review carefully before approving.

Also, this is not complete yet, I will be adding certain options as well.

### CODE REVIEW

**Review by amr-crabnebula — CHANGES_REQUESTED** [review]
just missing a change file

**Comment by naman-crabnebula** [comment]
> just missing a change file

Yeah, it is not complete yet.

**Review by lucasfernog-crabnebula — COMMENTED** [review]
Nice job, as long as we add the change file!

File: .changes/priority-and-section.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+"cargo-packager": minor
+---
+
+Add the support for priority and section in Debian Config
```
Comment (amr-crabnebula) [inline .changes/priority-and-section.md:5]:
```suggestion
Add `priority` and `section` options in Debian config
```

File: .changes/priority-and-section.md [hunk 2]
```diff
@@ -0,0 +1,5 @@
+---
+"cargo-packager": minor
```
Comment (amr-crabnebula) [inline .changes/priority-and-section.md:2]:
```suggestion
"cargo-packager": patch
"@crabnebula/packager": patch
```

**Review by amr-crabnebula — APPROVED** [review]

File: crates/packager/src/config/mod.rs [hunk 3]
```diff
@@ -358,6 +363,19 @@ impl DebianConfig {
...
+        self.section.replace(section.into());
+        self
+    }
+
+    /// Change the priority of the Debian Package. By default, it is set to `optional`.
+    /// Recognized Priorities as of now are :  `required`, `important`, `standard`, `optional`, `extra`
+    pub fn priority<S: Into<String>>(self, priority: S) -> Self {
+        self.priority.replace(priority.into());
+        self
+    }
```
Comment (amr-crabnebula) [inline crates/packager/src/config/mod.rs:377]:
```suggestion
    pub fn section<S: Into<String>>(mut self, section: S) -> Self {
        self.section.replace(section.into());
        self
    }

    /// Change the priority of the Debian Package. By default, it is set to `optional`.
    /// Recognized Priorities as of now are :  `required`, `important`, `standard`, `optional`, `extra`
    pub fn priority<S: Into<String>>(mut self, priority: S) -> Self {
        self.priority.replace(priority.into());
        self
    }
```

**Review by amr-crabnebula — APPROVED** [review]

**Pull request merged**